src/oscar/templates/oscar/dashboard/catalogue/category_form.html

```diff
@@ -23,10 +23,72 @@
 {% block headertext %}{{ title }}{% endblock %}
 
 {% block dashboard_content %}
-    <div class="table-header">
-        <h2>{{ title }}</h2>
-    </div>
-    <div class="well">
-        {% include 'oscar/dashboard/partials/form.html' with includes_files=1 class='wysiwyg' %}
-    </div>
+    <form action="{% if request.GET.urlencode %}?{{ request.GET.urlencode }}{% endif %}" method="post" class="form-stacked wysiwyg fixed-actions" enctype="multipart/form-data" data-behaviour="affix-nav-errors" autocomplete="off">
+        {% csrf_token %}
+        <div class="row">
+            {% block tab_nav %}
+                <div class="col-md-3">
+                    <div data-spy="affix" class="affix-top" data-offset-top="200">
+                        <div class="table-header">
+                            <h3>{% trans "Sections" %}</h3>
+                        </div>
+                        <ul class="nav nav-list bs-docs-sidenav" id="product_update_tabs">
```

Comment:
`id="product_update_tabs"` has been copied from the product form, but isn't semantically correct for a category form.